src/data_management/wms_layers/WmsLayerForm.tsx

```diff
@@ -0,0 +1,357 @@
+            console.error(data);
+          };
+        })
+        .catch((e:any) => console.error(e));
+      }
+  };
+
+  const {
+    values,
+    triedToSubmit,
+    tryToSubmitForm,
+    handleInputChange,
+    handleValueChange,
+    handleSubmit,
+    handleReset,
+    clearInput,
+  } = useForm({initialValues, onSubmit});
+
+  return (
+    <ExplainSideColumn
+      imgUrl={wmsIcon}
+      headerText={"WMS Layers"}
+      explainationText={"WMS-Layers allow to configure layers in lizard even if they are hosted on another platform"} 
+      backUrl={"/data_management"}
+    >
+      <form
+        className={formStyles.Form}
+        onSubmit={handleSubmit}
+        onReset={handleReset}
+      >
+        <span className={formStyles.FormFieldTitle}>
+          1: General
+        </span>
+        <TextInput
+          title={'Name *'}
+          name={'name'}
+          placeholder={'Please enter at least 3 characters'}
+          value={values.name as string}
+          valueChanged={handleInputChange}
+          clearInput={clearInput}
+          validated={!minLength(3, values.name as string)}
+          errorMessage={minLength(3, values.name as string)}
+          triedToSubmit={triedToSubmit}
+        />
+        <TextArea
+          title={'Description'}
+          name={'description'}
+          placeholder={'This is a layer based on raster_source'}
+          value={values.description as string}
+          valueChanged={handleInputChange}
+          clearInput={clearInput}
+          validated
+        />
+        <SlushBucket
+          title={'Datasets'}
+          name={'datasets'}
+          placeholder={'Search datasets'}
+          value={values.datasets as string[]}
+          choices={datasets.map((dataset: any) => {
+            return {
+              display: dataset.slug,
+              value: dataset.slug,
+            }
+          })}
+          valueChanged={(value: any) => {
+            console.log("datasets 123", value);
+            handleValueChange('datasets', value)
+          }}
+          validated
+        />
+        <span className={formStyles.FormFieldTitle}>
+          2: Data
+        </span>
+        <TextInput
+          title={'WMS Url *'}
+          name={'wms_url'}
+          placeholder={'http://example.com'}
+          value={values.wms_url as string}
+          valueChanged={handleInputChange}
+          clearInput={clearInput}
+          validated={!minLength(3, values.wms_url as string)}
+          errorMessage={minLength(3, values.wms_url as string)}
+          triedToSubmit={triedToSubmit}
+        />
+        <TextInput
+          title={'Slug *'}
+          name={'slug'}
+          placeholder={'Slug for this WMS'}
+          value={values.slug as string}
+          valueChanged={handleInputChange}
+          clearInput={clearInput}
+          validated={!minLength(3, values.slug as string)}
```

Comment:
Do we need at least 3 characters for the wms slug field

Reply:
Done, changed to 1 char (same as in old version)